panther_hardware_interfaces/test/unit/panther_system/test_lynx_system.cpp

```diff
@@ -0,0 +1,313 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cstdint>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <rclcpp/rclcpp.hpp>
+
+#include "panther_hardware_interfaces/panther_system/lynx_system.hpp"
+#include "panther_hardware_interfaces/panther_system/robot_driver/roboteq_robot_driver.hpp"
+
+#include "utils/system_test_utils.hpp"
+#include "utils/test_constants.hpp"
+
+class LynxSystemWrapper : public panther_hardware_interfaces::LynxSystem
+{
+public:
+  LynxSystemWrapper() : LynxSystem()
+  {
+    mock_robot_driver =
+      std::make_shared<::testing::NiceMock<panther_hardware_interfaces_test::MockRobotDriver>>();
```

Comment:
Why do the same like in the `test_roboteq_robot_driver.cpp`.


```suggestion
      std::make_shared<panther_hardware_interfaces_test::MockRobotDriver::NiceMock>();
```

Reply:
The same for `test_panther_system.cpp`